config/routes.rb

```diff
@@ -3,6 +3,9 @@
 
   root to: redirect('/topics', status: 302)
 
+  resources :step_by_step_pages, path: 'step-by-step-pages' do
```

Comment:
This generates a bunch of routes that we don't need yet.  Maybe limit this just to a get of the index page?